add-on/panels/js/desktopViews.jsx

```diff
@@ -0,0 +1,304 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var loop = loop || {};
+loop.shared = loop.shared || {};
+loop.shared.desktopViews = (function(_, mozL10n) {
```

Comment:
I think `_` doesn't get used in this file, so can you drop it from here and the end please.